kyuubi-rest-client/pom.xml

```diff
@@ -36,6 +36,66 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-common_${scala.binary.version}</artifactId>
```

Comment:
let us do not depend on kyuubi-common module, the kyuubi-rest-client module should independent